packages/expo-gl-cpp/cpp/UEXGL.h

```diff
@@ -76,6 +76,8 @@ void UEXGLContextMapObject(UEXGLContextId exglCtxId, UEXGLObjectId exglObjId, GL
 // [GL thread] Get the underlying OpenGL object an EXGL object maps to.
 GLuint UEXGLContextGetObject(UEXGLContextId exglCtxId, UEXGLObjectId exglObjId);
 
+void UEXGLInvalidateJsiCache();
```

Comment:
💅 `Jsi` or `JSI`? In some other place there's `ID`…

Reply:
I used ID for propNameIDCache because in jsi it's called jsi::PropNameID, in the rest of the code we are using `Id' e.g. `exglObjId`.
As for `jsi` I did not see any usages of jsi except namespace name `facebook::jsi` so I just followed convention we already used.